app/sidekiq/form526_submitted_email_job.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+require 'va_notify/service'
+
+class Form526SubmittedEmailJob
+  include Sidekiq::Job
+  include SentryLogging
+  sidekiq_options expires_in: 1.day
+
+  STATSD_ERROR_NAME = 'worker.form526_submitted_email.error'
+  STATSD_SUCCESS_NAME = 'worker.form526_submitted_email.success'
+
+  def perform(personalization_parameters)
+    @notify_client ||= VaNotify::Service.new(Settings.vanotify.services.va_gov.api_key)
+    @template_id ||= Settings.vanotify.services.va_gov.template_id.form526_submitted_email
```

Comment:
Interesting. You guys are using emails in the general va_gov area, ours are in the disability experiences area for some reason 🤔 

Ours should be in here probably, I am not sure why we have our own.